charts/civil-ccd/values.preview.template.yaml

```diff
@@ -1,5 +1,6 @@
 civil-service:
   java:
+    image: 'hmctspublic.azurecr.io/civil/service:pr-2724'
```

Comment:
For testing only, will be reverted before merge.

CCD will also need merged before civil